platform/core/src/lib.rs

```diff
@@ -5,5 +5,6 @@ mod feature_manifest_parsing;
 #[rustfmt::skip]
 #[allow(clippy::all)]
 #[allow(unused_imports)]
+#[allow(dead_code)]
```

Comment:
Shouldn't this be accompanied by a reason or something? So that you know when to remove it finally.

Reply:
Well, the reason is that it is generated code not controlled by us. I can add additional comments if it makes it clearer.

Reply:
Added explanation in a comment.